src/components/diagrams/singleLineDiagram/single-line-diagram-pane.js

```diff
@@ -174,24 +178,18 @@ export function SingleLineDiagramPane({
     visible,
 }) {
     const studyUpdatedForce = useSelector((state) => state.studyUpdated);
-
     const [updateSwitchMsg, setUpdateSwitchMsg] = useState('');
-
     const [views, setViews] = useState([]);
     const fullScreenSldId = useSelector((state) => state.fullScreenSldId);
-
     const [displayedSLD, setDisplayedSld] = useState([]);
     const [displayedSldHeights, setDisplayedSldHeights] = useState([]);
     const displayedSldHeightsRef = useRef();
     displayedSldHeightsRef.current = displayedSldHeights;
-
     const createView = useDisplayView(network, studyUuid, currentNode);
-
     const dispatch = useDispatch();
-
     const sldState = useSelector((state) => state.sldState);
-
     const viewsRef = useRef();
+    const { getNameOrId } = useNameOrId();
```

Comment:
seems not used, remove then ?

Reply:
it's used in the line 367 and 399

Reply:
my bad... the previous function had the same name